app/assets/stylesheets/local/index.scss

```diff
@@ -1,9 +1,147 @@
-.range_limit {
```

Comment:
So glad you asked. I meant to move out the couple of non-homepage things but never did and they ended up nested in the homepage wrapper. That's now fixed. Renamed the file to homepage.